variables/String from array of objects.tpl

```diff
@@ -159,10 +174,17 @@ if (!isArray(inputArray)) {
   return;
 }
 
-return inputArray
-  .map(obj => obj[keyToConcatenate])
-  .filter(obj => obj)
-  .join(delimiter);
+if (data.entireObject){
+  return inputArray
+    .map(obj => obj)
+    .join(delimiter);
+}
+else {
+  return inputArray
+    .map(obj => obj[keyToConcatenate])
+    .filter(obj => obj)
```

Comment:
These could be combined into a single

```
return inputArray
  .map(obj => data.entireObject ? obj : obj[keyToConcatenate])
  ...
```

but see my previous comment about should this variable handle different conversion types.